PyTest_POM_PoC/utils/driver_chrome.py

```diff
@@ -0,0 +1,5 @@
+from selenium import webdriver
```

Comment:
actually U don't need this file, bc U already defined fixture (conftest.py), which are doing the same